app/konnect/gateway-manager/provision-cloud-gateway.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Provision a Dedicated Cloud Gateway
+---
+
+This guide explains how to provision a [Dedicated Cloud Gateway](/konnect/dedicated-cloud-gateways/) and scale the data plane nodes in {{site.konnect_short_name}}.
+
+## Prerequisites
+
+* A network configured in {{site.konnect_short_name}} <!--I feel like a link is missing here, but I can't find a good doc to link to-->
+
+## Provision your fully-managed data plane nodes
+
+{% navtabs %}
+{% navtab UI %}
+1. From [Gateway Manager](https://cloud.konghq.com/gateway-manager) in the navigation menu, click the **New Control Plane** menu and select **{{site.base_gateway}}**.
```

Comment:
```suggestion
1. From [Gateway Manager](https://cloud.konghq.com/gateway-manager), click the **New Control Plane** button and select **{{site.base_gateway}}**.
```
If I see "the navigation menu", I assume that we're referring to the left side nav for Konnect, but this dropdown is directly on the page.

Reply:
@lena-larionova I was using "menu" in place of drop down here to follow Google's style guide: https://developers.google.com/style/word-list#drop-down. Personally, I agree with you that "menu" is kind of confusing for a drop down, so if we're okay with breaking with Google's guidance there, I can change this to drop down.

Reply:
Technically those are buttons with dropdown functionality, so we can just say "button"